architecture/additional_concepts/storage.adoc

```diff
@@ -254,6 +254,64 @@ take advantage of the volume's ability for ROX. Errors in the provider show up
 at runtime as mount errors.
 ====
 
+The table below lists the access modes supported by different persistent volumes.
+
+.Supported Access Modes for Persistent Volumes
+[cols=",^v,^v,^v", width="100%",options="header"]
+|===
+|Volume Plugin  |ReadWriteOnce  |ReadOnlyMany  |ReadWriteMany  
+|AWS EBS  | X | - |  -
+|Azure Disk | X | - | - 
+|Ceph RBD  | X | X |  -
+|Fiber Channel  | X | X |  -  
+|GCE Persistent Disk  | X | - |  -
+|GlusterFS  | X | X |  X
+|HostPath  | X | - |  -
+|iSCSI  | X | X |  -
+|NFS  | X | X | X 
+|Openstack Cinder  | X | - |  -
+|===
+
+[NOTE]
+====
+ * Please use link:https://docs.openshift.com/container-platform/latest/dev_guide/deployments/deployment_strategies.html#recreate-strategy[Recreate Deployment Strategy], if pods rely on AWS EBS, GCE Persistent Disks, or Openstack Cinder PVs.
+  * Azure Disk do not support dynamic provisioning.
+====
+
+ifdef::openshift-dedicated[]
+Following restrictions apply when using persistent volumes with OpenShift Dedicated.
+
+[IMPORTANT]
+====
+ . Pvs are provisioned with either EBS volumes (AWS) or GCP storage (GCP), depending on where the cluster is provisioned.
+ . Only RWO access access mode is applicable, since EBS volumes and GCE Persistent Disks cannot be mounted to to multiple nodes.
+ . emptyDir has the same lifecycle as the pod:
+   * emptyDir volumes survive container crashes/restarts.
+   * emptyDir volumes are deleted when the pod is deleted.
```

Comment:
I see what you mean about the dot points now. I'd avoid using numbered lists unless it's a procedure, so I think using bullets is ideal.

You can nest lists by doubling up on the asterisks:

* blah
** blah